packages/table/src/locator.ts

```diff
@@ -24,12 +24,26 @@ export interface ILocator {
     convertPointToColumn: (clientX: number) => number;
 
     /**
+     * Locates a column boundary given the client X coordinate. The boundary
+     * is returned as the index of the column appearing to the right of that
+     * boundary. Returns -1 if the coordinate is not over a column.
+     */
+    convertPointToColumnLeftBoundary: (clientX: number) => number;
```

Comment:
i feel like the word "boundary" is wrong here. could this be simply `convertPointToColumnIndex`?